nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config

```diff
@@ -58,6 +58,11 @@ profiles {
   test { includeConfig 'conf/test.config' }
 }
 
+// Avoid this error:
+// WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
+// Thanks to: https://github.com/alesssia/YAMP/wiki/How-to-use-Docker
+docker.runOptions = '-u $(id -u):$(id -g)'
```

Comment:
Shouldn't it be `-u \$(id -u):\$(id -g)`?
cf: https://github.com/nf-core/sarek/blob/10eff4e8ff71e672d73a3bf4aa43a3693342bfb2/nextflow.config#L63

Reply:
Not sure that is required - the `'...'` should be enough if it's in `nextflow.config` right?

Reply:
sorry, I didn't understood what you meant ;-)

Reply:
I mean I'm not sure whether we need to escape the `$` with a `\$` here as they are inside a `'...'` single quotes block :)

Reply:
Aaaah, now I understand.
I'm pretty sure I copied that from someone else (probably Paolo) at some point.
I'm guessing the double quotes render the escaping the `$` a necessity

Reply:
Then we need to address this as well - I was looking for another place where this is used but couldn't find it. Wouldn't `fixOwnership` as listed here do the same? https://www.nextflow.io/docs/latest/config.html#config-docker